setup.py

```diff
@@ -33,26 +32,27 @@
           'Topic :: Scientific/Engineering',
           'License :: OSI Approved :: BSD License',
 
-          'Programming Language :: Python :: 3.6',
           'Programming Language :: Python :: 3.7',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
+          'Programming Language :: Python :: 3.10',
       ],
 
       scripts=['scripts/trainN2V.py',
-          'scripts/predictN2V.py'
-      ],
+               'scripts/predictN2V.py'
+               ],
 
       install_requires=[
           "numpy",
-          "scipy",
-          "matplotlib",
-          "six",
-          "keras>=2.1.1,<2.4.0",
-          "tifffile>=2020.5.11",
+          "tifffile",
           "imagecodecs>=2020.2.18",
-          "tqdm",
           "backports.tempfile;python_version<'3.4'",
-          "csbdeep>=0.6.0,<0.7.0",
+          "csbdeep>=0.7.2",
```

Comment:
Can we pin it to be <0.8.0?

Reply:
Yes that would make sense. Done in https://github.com/juglab/n2v/pull/127/commits/63412f03fa60ec68e4acb5e92e108391c3d6e787.